pkg/build.ml

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ocaml 
+#directory "pkg"
+#use "topkg.ml"
+
+let () = 
+  Pkg.describe "higher" ~builder:(`OCamlbuild []) ([
```

Comment:
You can use ```~builder:(`OCamlbuild_no_ocamlfind [])``` if you want to drop the dependency on `ocamlfind`. This can be done with packages that do not depend on other packages.